talend-component-maven-plugin/src/it/web/.jenkins/scripts/tcomp-build.sh

```diff
@@ -61,6 +61,7 @@ main() (
   --projects \!org.talend.sdk.component:credentials-component \
   --projects \!org.talend.sdk.component:datastorevalidation-component \
   --projects \!org.talend.sdk.component:dropdownlist-component \
+  --projects \!org.talend.sdk.component:documentation \
```

Comment:
What's the goal of this script? skip some modules building? Where is it used?

Reply:
It is used in tck API test to have the quickest build to run test on it.
https://github.com/Talend/component-runtime/blob/a63271daab9dbadb82b864d70f3580a9e45bd821/talend-component-maven-plugin/src/it/web/.jenkins/Jenkinsfile#L263

Reply:
We may run those test during main build in near future.

Reply:
Reverted on this PR because optimized in https://github.com/Talend/component-runtime/pull/696